dask/array/core.py

```diff
@@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
     return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
 
 
+def from_value(value, shape, dtype=None, name=None):
```

Comment:
Yeah, I think `from_imperative` is probably the best choice. A little long for my taste, but it's pretty clear `value` is too ambiguous to be a good name.